ChangeLog

```diff
@@ -1,16 +1,14 @@
 CHANGES
 =======
 
```

Comment:
it seems odd to get rid of changes in the changelog? not sure what's happening here.

Reply:
The ChangeLog is autogenerated by pbr based on the git commit history. When stuff is removed, it should be because those commits are part of the master branch history. Sometimes this happens during messy merges.